Api/Data/SourcesInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Ampersand\DisableStockReservation\Api\Data;
+
+/**
+ * Interface SourcesInterface
+ * @package Ampersand\DisableStockReservation\Api\Data
+ */
+interface SourcesInterface
+{
```

Comment:
So something I usually do with an interface (and so does Magento core) is to also define the properties of the object as constants. That way they're defined in a single location and we can reference them throughout the functionality. So:
```php
/**
 * Order ID key
 */
const ORDER_ID_KEY = 'order_id';
```